addons/it/sdi/extensions.go

```diff
@@ -15,6 +15,7 @@ const (
 	ExtKeyExempt       cbc.Key = "it-sdi-exempt"
 	ExtKeyRetained     cbc.Key = "it-sdi-retained"
 	ExtKeyPaymentMeans cbc.Key = "it-sdi-payment-means"
+	ExtKeyVATCollect   cbc.Key = "it-sdi-vat-collect"
```

Comment:
I'm not super-convinced on "collect" as its a verb... but can't think of anything more concise...

Reply:
Just checking this in ChatGTP, the response is:
> “Esigibilità dell’IVA” translates to “VAT enforceability” or “VAT liability.” It refers to the moment when VAT becomes due or payable to the tax authorities.

Reply:
Addressed on 7b87944